ocaml/xapi/xapi_clustering.ml

```diff
@@ -545,6 +545,13 @@ module Watcher = struct
      is an update. *)
   let cluster_change_interval = Mtime.Span.(5 * min)
 
+  (* CA-396635: Sometimes it takes the underlying cluster stack (corosync) some time
+     to return a consistent view of the quorum. For example, it may be that the membership
+     information correctly reflects the new members after a membership change, while the
+     quorum field is still out of date. Add a delay here to make sure that the information
+     from corosync represents a consistent snapshot of the current cluster state. *)
```

Comment:
Can't the consistent snapshot be waited for instead of a delay?

Reply:
there is no way to know when the consistent snapshot would be ready though.... at least I am not aware of any corosync apis that can do that

Reply:
> Add a delay here to make sure that the information from corosync represents a consistent snapshot of the current cluster state.

So the delay will not make sure about also? So I can assume it will make most of the cases better.

Reply:
Yes. There is no way to guarantee a consistent one as far as I am aware. But the delay should make things better.